.github/workflows/validate-pr.yml

```diff
@@ -133,6 +133,13 @@ jobs:
           path: ${{ env.pythonLocation }}
           key: ${{ runner.os }}-${{ env.pythonLocation }}-pip-${{ hashFiles('substrafl/setup.py') }}-${{ hashFiles  ('substrafl/docs/requirements.txt') }}-${{ hashFiles('substrafl/benchmark/camelyon/requirements.txt') }}
 
+      - uses: actions/cache@v3.0.8
+        id: data
+        with:
+          path: /home/runner/work/substrafl/data
+          key: data
+          # The benchmark data cache has to be updated manually if the data changes
```

Comment:
What would be a possible way to be notify if the data changes ?

Reply:
And how do we update ? Could be worth to explain it in the benchmark readme

Reply:
> What would be a possible way to be notify if the data changes ?

the workflow fails because the URL changes

> And how do we update ? Could be worth to explain it in the benchmark readme

change the key or use github API, easy to find in the action documentation